test/e2e/resources/components/AdvancedWidget.xml

```diff
@@ -1,7 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component name="AdvancedWidget" extends="NormalWidget">
-    <script type="text/brightscript" uri="pkg:/components/scripts/AdvancedWidget.brs" />
```

Comment:
For this test I only want to test children and fields but not scripts. Scripts and init are tested in a different set of components.

Reply:
Great idea!  That might be worth putting in an XML comment too :)